coralproject/talk
# Feat/corl 2230 configurable flatten replies

## What does this PR do?
This PR addresses https://vmproduct.atlassian.net/browse/CORL-2230 and makes "Flatten Replies" configurable via the admin UI, as opposed to a feature flag.

## What changes to the GraphQL/Database Schema does this PR introduce?
FLATTEN_REPLIES is no longer a valid feature flag.

## How do I test this PR?
1. Create a reply chain that is long enough to be hidden behind a "Show more of this conversation" button. 
2. Under admin > configure > general, navigate to Flatten Replies and check "Yes"
3. back in the stream, refresh and confirm that all replies are shown.
 
## How do we deploy this PR?
No special actions should be necessary to deploy this change NOTE: Need to confirm this wont break tenants/instances where flatten replies are enabled as a feature flag / if we need some sort of migration.
